lb.tf

```diff
@@ -84,13 +84,13 @@ resource "aws_lb_target_group" "default" {
   }
 
   health_check {
-    interval            = 30
+    interval            = var.health_check.interval
     timeout             = var.protocol != "TCP" ? 3 : null
     protocol            = var.protocol
-    path                = var.protocol != "TCP" ? var.health_check_path : null
+    path                = var.protocol != "TCP" ? var.health_check.path : null
```

Comment:
Hey @justindotcloud, I already approved the change but when having a closer look, can this be a `null` value? Will it still work with `path = null`?

Reply:
Yes... You can set the `path` van in the object to `null` (which it is by default). So should behave the same as before.

Reply:
Great 👍